modules/apps/site/site-insurance-site-initializer/remote-web-components/raylife-d2c-web/src/components/containers/Forms/Business.js

```diff
@@ -80,6 +81,7 @@ export const FormBusiness = () => {
           label="Do you sell products under your own brand or label?"
           rules={{ required: true }}
           control={control}
+          defaultValue={false}
```

Comment:
But here, `defaultValue` is **false** as boolean.

That's the idea, or should everything be a **string** or **boolean**?

Reply:
This should be a string...

Reply:
I put `false`, because following the mock, this field doesn't have a default value. When a remove this parameter the field stay with "no" selected by default. How can I do it right? ty.